### PR TITLE
Html overhaul

### PR DESCRIPTION
**Pull request by lars-radtke**
I undertook my project a complete overhaul.

Since the task was to create a one-pager from a multi-page site, the index.html was restructured.

Also due to the introduction of JavaScript to the project a lot of different/new CSS classes have been created.

Not all of the JavaScript functions work properly yet; most prominent example is when you bookmark questions, then toggle "show bookmarked only ON" and then unbookmark a question, funky things happen. I haven't solved that issue ye but get to it ASAP.

### CODE REVIEW

File: resources/css/components/header.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	text-align: center;
+	color: rgb(var(--white));
+	cursor: default;
+}
+
+.backdrop {
+	background-color: rgba(var(--blue-sapphire),0.5);
+}
+
+@supports (backdrop-filter: blur()) {
```
Comment (lars-radtke) [inline resources/css/components/header.css]:
```suggestion
@supports (backdrop-filter: blur(4px)) {
```

File: resources/css/components/header.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	top: 0;
+	left: 0;
+	text-align: center;
+	color: rgb(var(--white));
+	cursor: default;
+}
+
+.backdrop {
+	background-color: rgba(var(--blue-sapphire),0.5);
+}
```
Comment (lars-radtke) [inline resources/css/components/header.css:17]:
```suggestion
    z-index: 1;
}
```

File: resources/css/components/header.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	cursor: default;
+}
+
+.backdrop {
+	background-color: rgba(var(--blue-sapphire),0.5);
+    z-index: 1;
+}
+
+@supports (backdrop-filter: blur(4px)) {
+	.header {
```
Comment (lars-radtke) [inline resources/css/components/header.css]:
```suggestion
	.backdrop {
	-webkit-backdrop-filter: blur(4px);
```

File: index.html [hunk 4]
```diff
@@ -1,180 +1,131 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+	<title>sQUIZd | start</title>
+	<link rel="stylesheet" type="text/css" href="resources/css/styles.css">
+	
   </head>
   <body>
-    <header class="header">
-      <h1><span>s</span>QUIZ<span>d</span></h1>
+    <header class="header backdrop flex f-column f-justify-between">
+      <h1 class="h1">
```
Comment (marcbendixen) [inline index.html:13]: Small hint: `h1`, `h2` and so on, aren't a good choices for class names. Better way would be `headline`, `main-heading`, `subtitle`  or anything like that.

File: index.html [hunk 5]
```diff
@@ -1,180 +1,131 @@
...
+    		    class="footer__button"
+    		    id="footer__button--profile"
+				title="Go to your profile"
+    		    aria-label="Go to your profile"
+			>
+			<img class="footer__icon" src="resources/icons/captain.svg">
+			</button>
+	</div>
     </footer>
+	<script src="resources/js/scripts.js"></script>
```
Comment (marcbendixen) [inline index.html:129]: Very small hint and nothing special 😉 naming convention for this folder would be `src/...`. You will encounter this name more often in the future.

File: resources/css/components/body.css [hunk 6]
```diff
@@ -0,0 +1,14 @@
+* {
+	font-family: "Montserrat", sans-serif;
+	line-height: 1.3;
+	scroll-behavior: smooth;
```
Comment (marcbendixen) [inline resources/css/components/body.css:4]: Good job! One of the first things I always add to projects is the `scroll-behavior` 😉 👍

File: resources/css/components/tile.css [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+	font-variation-settings: "wght" 600;
+}
+
+.tile__bookmark {
+	position: absolute;
+	top:10px;
+	right:10px;
+	height: 50px;
+	width: 50px;
+	background-image: url(../../icons/bookmark.svg);
```
Comment (marcbendixen) [inline resources/css/components/tile.css:39]: Small hint: there is a shorthand to group background-settings. For example: `background: no-repeat url(../../icons/bookmark.svg);`

You can find more information [here](https://developer.mozilla.org/en-US/docs/Web/CSS/background).

File: resources/js/scripts.js [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+let answerButtons = document.querySelectorAll(".tile__answer-button")
+answerButtons.forEach(function(i){
+	i.addEventListener("click", () => {
+		i.nextSibling.classList.toggle("tile__answer--hidden")
+})})
+
+BUTTON_START.onclick = buttonStart
+BUTTON_CREATE.onclick = buttonCreate
+BUTTON_PROFILE.onclick = buttonProfile
```
Comment (marcbendixen) [inline resources/js/scripts.js:74]: In a normal environment we don't use `SCREAMING_SNAKE_CASE` as `const`/variable names. Instead `camelCase` is the way to go and what you'll see everywhere 😉

**Review by marcbendixen — COMMENTED** [review]
Well done 👍 
Just some small hints, nothing special.
Keep up the good work.

File: index.html (hunk 5 above)
Comment (lars-radtke) [reply index.html:129]: Ah! Okay, i didn't know about that. Thanks, i will change to that in the future.

File: index.html (hunk 4 above)
Comment (lars-radtke) [reply index.html:13]: I can see that. I'm going to address that in the next push.

File: resources/js/scripts.js (hunk 8 above)
Comment (lars-radtke) [reply resources/js/scripts.js:74]: Ay! I read over our protocol again and seemingly misunderstood the definition of hardcoded constants. As my scripts need a little work anyway, I'm going to change the names of the constants on the go. Thanks for all your input!

**Pull request merged**